src/schemas/space.json

```diff
@@ -171,6 +171,14 @@
         },
         "plugins": {
           "type": "object"
+        },
+        "votingDelay": {
```

Comment:
Can you make "voting" a root param with delay and period as sub param like this:

{
  "voting": {
    "delay": 1651651,
    "period": 894984
  }
}